src/librustc/middle/cstore.rs

```diff
@@ -67,6 +67,8 @@ pub struct CrateSource {
 
 #[derive(RustcEncodable, RustcDecodable, Copy, Clone, Ord, PartialOrd, Eq, PartialEq, Debug)]
 pub enum DepKind {
+    /// A dependency that is only used for its macros, none of which are visible from other crates.
+    UnexportedMacrosOnly,
```

Comment:
Are these only used by monomorphised functions or is there some reason to include them. Would be good to comment in any case.